Master and node can't connect  via IPv6.

**Describe the bug**
Master has IPv4 And IPv6.Some nodes only have IPv6.
Master and node can't connect  via IPv6.
IPv4 to IPv4 is ok.
---/etc/munin/munin.conf---In master
> [Group3;gitlab-server]
address AAAA domain record
use_node_name yes

---/var/log/munin-node/---In node
> host_name Gitlab
allow IPv6 Address
host *

**To Reproduce**
Steps to reproduce the behavior:
1 Settings ipv6 connect 
2.`$ munin-update --screen
2021-05-27 02:27:46 [critical][188806]: [FATAL] Socket read from pbx-ipv6 failed.  Terminating process.
2021-05-27 02:27:46 [critical][188809]: [FATAL] Socket read from gitlab-ipv6 failed.  Terminating process.
`
3c.And no log in /var/log/munin

4.telnet is ok
```
telnet gitlab-ipv6.memorz.fun 4949
Trying ipv6:5054:ff:fe48:f3bc...
Connected to gitlab-ipv6
Escape character is '^]'.
# munin node at Gitlab
1
# Unknown command. Try cap, list, nodes, config, fetch, version or quit
1
# Unknown command. Try cap, list, nodes, config, fetch, version or quit
cap
cap multigraph dirtyconfig
list
nodes
gitlab
.
config
# Unknown service
.
fetch
# Unknown service
.
version
munins node on Gitlab version: 2.0.67
list

list all

^C^C^C^\^C^\^]
telnet> a
?Invalid command
telnet> 1
?Invalid command
telnet> 2
?Invalid command
telnet> Connection closed.
```
And a new log has been generated
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots & Logs**
If applicable, please add screenshots and/or logs to help explain your problem.
![20210527025214](https://user-images.githubusercontent.com/14276116/119715683-d8680680-be96-11eb-8d90-55cf6bb276c2.png)
![master-config](https://user-images.githubusercontent.com/14276116/119715492-a5be0e00-be96-11eb-80a3-15bbdce165c2.png)
![node-config](https://user-images.githubusercontent.com/14276116/119715498-a656a480-be96-11eb-96fe-5e3cb5618f25.png)


**Desktop (please complete the following information):**
 - Debian bullseye/munin from experiment repo
 - firefox
 - 2.999.15
![node-config](https://user-images.githubusercontent.com/14276116/119715119-39dba580-be96-11eb-87a6-86650b775525.png)
![dig](https://user-images.githubusercontent.com/14276116/119715913-1e24cf00-be97-11eb-95d1-b1e433ec2380.png)


**Additional context**
Add any other context about the problem here.

Today I found an issue where munin could no longer connect to my IPv6-only munin-nodes after applying that IO::Socket::INET6 => IO::Socket::IP change from
https://github.com/munin-monitoring/munin/commit/012b33a7#diff-bb268ce7158f5da204a7ac5e33a4d64505a31504c00061c35eee39d16581ae98

When using a DNS-name that has both IPv6 and IPv4, it will connect over IPv4.
When using the kilo6.zq1.de DNS-name that has only IPv6, it will not connect.

`/var/log/munin/munin-update.log` had
```
2025/07/25 06:32:34 [FATAL] Socket read from kilo.zq1.de failed.  Terminating process. at /usr/lib/perl5/vendor_perl/5.26.1/Munin/Master/UpdateWorker.pm line 266.
2025/07/25 06:32:34 [ERROR] Munin::Master::UpdateWorker<zq1.de;kilo.zq1.de> died with '[FATAL] Socket read from kilo.zq1.de failed.  Terminating process. at /usr/lib/perl5/vendor_perl/5.26.1/Munin/Master/UpdateWorker.pm line 266.'
2025/07/25 06:32:39 [INFO] Reaping Munin::Master::UpdateWorker<zq1.de;kilo.zq1.de>.  Exit value/signal: 18/0
```

This is on openSUSE Leap 15.6 with perl-5.26.1

--- a/lib/Munin/Master/Config.pm
+++ b/lib/Munin/Master/Config.pm
@@ -153,7 +153,7 @@ my %booleans = map {$_ => 1} qw(
 		graph_data_size  => 'normal',
 		graph_strategy   => 'cron',
 		groups           => {},
-		local_address    => 0,
+		local_address    => '::',
 		logdir           => $Munin::Common::Defaults::MUNIN_LOGDIR,
 		logoutput        => 'syslog',
 		max_processes    => 16,

